README.md

```diff
@@ -6,57 +6,91 @@ This is the GitHub repository of the [OGC Environmental Data Retrieval API Stand
 
 The **OGC API - Environmental Data Retrieval** candidate standard is part of the OGC API suite of standards. [OGC API standards](https://ogcapi.ogc.org) define modular API building blocks to spatially enable Web APIs in a consistent way. [OpenAPI](http://openapis.org) is used to define the reusable API building blocks.
 
-**The public comment period on the EDR API closed September 28th.** [Go here for more information.](https://www.ogc.org/standards/requests/215)
+### EDR API Query Patterns
+The EDR API allows a query, constructed of a spatio-temporal pattern, to retrieve data just for that pattern, from a data collection resource. The main patterns are:
+- **1 - Position**: Retrieve data for point/position for several parameters at a time instant, or as a timeseries, or as a vertical profile for a time instant.
+- **2 - Area**:  Retrieve data within a polygon or rectangular tile/subset at a time instant, or as a timeseries, or as a vertical profile for a time instant.
+- **3 - Trajectory and Corridor**: Retrieve data along a 2D, 3D or 4D trajectory or within a defined corridor around a trajectory.
 
-**A public Hackathon/Sprint** was held [virtually in March 2020](https://github.com/opengeospatial/EDR-API-Sprint) and another was held [9-10 November 2020](https://github.com/opengeospatial/OGCAPI-EDR-Sprint2) to help finalise the specification. There was a **public Webinar** outlining the Sprint's objectives on Wednesday 4 November 2020.
+Each of these patterns is optional, but a compliant API should implement at least one of them.
```

Comment:
I think this statement should be moved before any of the query patterns are listed (see above)

Reply:
Done